acpreprocessing/stitching_modules/nglink/write_nglink.py

```diff
@@ -32,3 +33,19 @@ def write_tinyurl(outputDir, state, fname):
     io.save_file(ff, url)
     print("Done! Neuroglancer Link:")
     print(url)
+
+
+def create_viz_link_from_json(
+    ngl_json,
+    outputDir,
+    fname,
+    url="https://json.neurodata.io/v1",
+    neuroglancer_link="http://bigkahuna.corp.alleninstitute.org/neuroglancer/#!"
+):
+    r = requests.post(url, json=ngl_json)
+    json_url = r.json()["uri"]
+    viz_link = f"{neuroglancer_link}{json_url}"
+    ff = os.path.join(outputDir, fname)
```

Comment:
can "generate and save a link" be a separate method?  I think there are use cases for generating and returning the link (similar to just generating a tinyurl as seen in the tutorial about links).

Reply:
I think it would also be good to add an option for all positions to be within one layer. Looksl ike there is a strong use case for that right now .